src/schemas/fairdata-ida.ui.js

```diff
@@ -710,6 +710,15 @@ export default {
 			'count': 100,
 			'grouped': false,
 			'labelNameInSchema': 'title',
+			'defaultValue': [{
+				"identifier":"http://uri.suomi.fi/codelist/fairdata/license/code/CC-BY-4.0",
+				"label": {
+					"sv": '',
```

Comment:
Any specific reason to keep this empty or we can have same values as others have irrespective of language translation